src/viam/sdk/config/viam-cpp-sdk-libviamsdk.pc.in

```diff
@@ -1,12 +1,11 @@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix=/usr
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@/viam/sdk
 
 Name: @PROJECT_NAME@-libviamsdk
 Description: @PROJECT_DESCRIPTION@
 URL: @PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
-Requires: gRPC++ >= @VIAMCPPSDK_GRPCXX_VERSION@ protobuf >= @VIAMCPPSDK_PROTOBUF_VERSION@ @PROJECT_NAME@-libviamapi >= @PROJECT_VERSION@
```

Comment:
- As above, I don't think these `Requires` can or should be removed.
- I think this constraint on the `libviamapi` version should remain as well, but looking more closely it looks slightly wrong, in that it doesn't exactly match the name at [src/viam/api/config/viam-cpp-sdk-libviamapi.pc.in](https://github.com/viamrobotics/viam-cpp-sdk/pull/165/files#diff-ef864422f69d067ff0260bbbfd3e070ad5ac15756200c737f33ff5dd49437ae2). I think maybe that `.pc.in` should have `Name: @PROJECT_NAME@-lib**viam**api` instead.